crates/hir_def/src/nameres/tests/mod_resolution.rs

```diff
@@ -323,13 +323,26 @@ pub struct Baz;
 fn module_resolution_relative_path_outside_root() {
     check(
         r#"
-//- /main.rs
+//- /a/b/c/d/e/main.rs root:
```

Comment:
```suggestion
//- /a/b/c/d/e/main.rs crate:main
```

I think this should do the same without introducnig code for `explicit_root` handling

Reply:
That worked.  I'll revert the explicit root stuff.